scriptmodules/supplementary/bluetooth.sh

```diff
@@ -402,6 +404,8 @@ function gui_bluetooth() {
                     ;;
             esac
         else
+            # restart sixad (if running)
+            service sixad status >/dev/null && service sixad restart
```

Comment:
Yes, maybe it is. If I reboot the system and invoke "sudo sixaxis -r" before pairing, the device will not pair correctly. It's probably better to restart the daemon so the stack doesn't interfere.